src/pages/index.js

```diff
@@ -48,52 +78,114 @@ const cardListEl = new Section(
 cardListEl.renderItems();
 
 const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  descriptionSelector: ".profile__description",
+  title: ".profile__title",
+  description: ".profile__description",
+  avatarSelector: ".profile__image",
 });
 
+api
+  .getUserInfo()
+  .then((userData) => {
+    userInfo.setUserInfo({
+      title: userData.name,
+      description: userData.about,
+      avatar: userData.avatar,
+    });
+  })
+  .catch((err) => console.error(err));
+
 const cardImageModal = new PopupWithImage("#preview-image-modal");
 cardImageModal.setEventListeners();
 
 function handleImageClick(cardData) {
   cardImageModal.open(cardData);
 }
 
-const addCardPopup = new PopupWithForm(
-  "#add-card-modal",
-  handleAddCardFormSubmit
+const deleteModalConfirmation = new PopupConfirmation(
+  "#modal-confirm",
+  async (cardId, cardElement) => {
+    try {
+      await api.deleteCard(cardId);
+      cardElement.remove();
+    } catch (err) {
+      console.error(`Error On Card Deletion ${err}`);
+    }
+  }
 );
-addCardPopup.setEventListeners();
+
+function handleDeleteCard(card) {
+  deleteModalConfirmation.setSubmitAction(() => {
+    api
+      .deleteCard(card.getId())
+      .then(() => {
+        card.remove();
+        deleteModalConfirmation.close();
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  });
+  deleteModalConfirmation.open();
+}
+
+function handleLikeClick(card) {
+  if (card._setIsLiked) {
```

Comment:
`_setIsLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.